src/PL/Password/Generator/Option.java

```diff
@@ -0,0 +1,151 @@
+package PL.Password.Generator;
+
+import java.awt.*;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+import static PL.Password.Generator.CommonlyMethods.clearConsole;
+import static PL.Password.Generator.CommonlyMethods.isaBoolean1to2;
+import static PL.Password.Generator.Program.start;
+import static PL.Password.Generator.Tab.*;
+
+
+public class Option {
```

Comment:
`ChosenOptionHandler` maybe? Or sth similar?
Also this class seems to be too large, please think of splitting it into smaller classes.
Please read about SOLID principles.